cpp/src/groupby/hash/groupby.cu

```diff
@@ -594,22 +590,21 @@ std::unique_ptr<table> groupby_null_templated(table_view const& keys,
                                               rmm::cuda_stream_view stream,
                                               rmm::mr::device_memory_resource* mr)
 {
-  auto d_keys = table_device_view::create(keys, stream);
-  auto map    = create_hash_map<keys_have_nulls>(*d_keys, include_null_keys, stream);
+  auto d_keys   = table_device_view::create(keys, stream);
```

Comment:
Typically I prefer to have the suffix `_ptr` for the result output from `*_device_view::create()` because it is a (smart) pointer.

Reply:
:+1: